adapters/krushmedia/krushmediatest/exemplary/banner-app.json

```diff
@@ -0,0 +1,156 @@
+{
+    "mockBidRequest": {
+      "id": "some-request-id",
+      "device": {
+        "ua": "test-user-agent",
+        "ip": "123.123.123.123",
+        "language": "en",
+        "dnt": 0
+      },
+      "tmax": 1000,
+      "user": {
+        "buyeruid": "0000-000-000-0000"
+      },
+      "app": {
+        "publisher": {
+          "id": "123456789"
+        },
+        "cat": [
+          "IAB22-1"
+        ],
+        "bundle": "com.app.awesome",
+        "name": "Awesome App",
+        "domain": "awesomeapp.com",
+        "id": "123456789"
+      },
+      "imp": [
+        {
+          "id": "some-impression-id",
+          
+          "banner": {
+            "w":320,
+            "h":50
+          },
+          "ext": {
+            "bidder": {
+              "host": "example.com",
+              "sourceid": "partner",
+              "accountid": "hash"
```

Comment:
Since the JSON annotation for account id is `key` here: https://github.com/prebid/prebid-server/pull/1504/files#diff-724b5d286e85be13b490de0a88acfd4dR5 you need to use `key` here rather than `accountid`. This wasn't caught as an error because, 1. You are missing the macro in the test endpoint as mentioned above and 2. You need to update the `uri` in the `expectedRequest` below to have the account ID value mentioned in this bidder ext here.

Also, any reason why you have `sourceid` and `host` defined here? From the `ExtKrushmedia` struct it looks like you only support account ID

Reply:
Looks like you have it defined right in most of the tests. Can you please update it for the two tests that don't have it rightly defined and update the test endpoint have a Account ID macro?